src/pages/en/guides/markdown-content.md

```diff
@@ -459,6 +460,28 @@ export default {
 }
 ```
 
+#### Heading IDs and plugins
+
+Astro injects an `id` attribute into all heading elements (`<h1>` to `<h6>`) in Markdown and MDX files and provides a `getHeadings()` utility for retrieving these IDs in [Markdown exported properties](#exported-properties).
+
+You can customize these heading IDs by adding a rehype plugin that injects `id` attributes (e.g. `rehype-slug`). Your custom IDs, instead of Astro's defaults, will be reflected in HTML output and the items returned by `getHeadings()`.
```

Comment:
```suggestion
You can customize these heading IDs by adding a rehype plugin that injects `id` attributes (e.g. `rehype-slug`). Your custom IDs, instead of Astro's defaults, will be reflected in the HTML output and the items returned by `getHeadings()`.
```

Having a brain freeze, but do we need a "the" here?

Reply:
I _think_ it’s one of those situations where it doesn’t really matter (?) but sure, why not 👍